tests/common/nebula_service.py

```diff
@@ -85,7 +85,7 @@ def _telnet_port(self, port):
     def install(self):
         if os.path.exists(self.work_dir):
             shutil.rmtree(self.work_dir)
-        os.mkdir(self.work_dir)
+        os.makedirs(self.work_dir)
```

Comment:
Don't do this, it's feature that don't allow to create server if `build` folder don't exists.

Reply:
So your path generation is wrong, you created it with the current directory, not the build directory as the parent node. you can make it as @jievince said.

Reply:
Yes, build_dir better.